diffrax/integrate.py

```diff
@@ -38,6 +40,77 @@
 from .term import AbstractTerm, VectorFieldWrapper, WrapTerm
 
 
+class _HistoryVectorField(eqx.Module):
+    vector_field: Callable
+    t0: float
+    tprev: float
+    tnext: float
+    dense_info: PyTree[Array]
+    dense_interp: Optional[DenseInterpolation]
+    interpolation_cls: Type[AbstractGlobalInterpolation]
```

Comment:
I think this should have type `Type[AbstractLocalInterpolation]`?

Reply:
FWIW these type annotations aren't actually checked yet. They will be soon-ish, once some of the current Diffrax changes have happened and I can upgrade this codebase to use [jaxtyping](https://github.com/google/jaxtyping).

Reply:
yes you are right since we only need value between `state.tprev` and `state.tnext` and not all `dense_ts`